checker/src/main.rs

```diff
@@ -42,69 +43,74 @@ fn main() {
         env_logger::init_from_env(e);
     }
 
-    // Get MIRAI options from environment variable. For now, we do not pass them
-    // on the command line because `RUSTC_WRAPPER=mirai cargo build` does not accept any
-    // options and arguments which it does not known about (even though we could remove them in
-    // this code here and not pass on to rustc). Once we have cargo integration
-    // of MIRAI we may want to change this.
+    // Get any options specified via the MIRAI_FLAGS environment variable
```

Comment:
Have you tried whether this actually worked if MIRAI is called via  RUSTC_WRAPPER=mirai cargo build as described in the comment you deleted? For me this actually failed because cargo doesn't seem to allow passing any arguments it doesn't know about. That's why I did not follow this pass in the first place. But experiment I made may have been faulty, so...

Reply:
You are correct that cargo does not pass through command line arguments to rustc when building. For that something like MIRAI_FLAGS is needed and I have not changed anything there. This PR just adds the possibility of using mirai directly and passing in command line arguments as one would expect.

Reply:
Great, just wanted to be sure that it's clear this doesn't work. 

If we look at pure MIRAI command line (which we may want to use for creating `cargo mirai`), I'm not sure whether we need any more than `<mirai_args> -- <rustc_args>`, but good to have it now anyway.